pkg/util/kube/security_context_test.go

```diff
@@ -25,32 +25,43 @@ import (
 
 func TestParseSecurityContext(t *testing.T) {
 	type args struct {
-		runAsUser  string
-		runAsGroup string
+		runAsUser                string
+		runAsGroup               string
+		allowPrivilegeEscalation string
 	}
 	tests := []struct {
 		name     string
 		args     args
 		wantErr  bool
 		expected *corev1.SecurityContext
 	}{
-		{"valid security context", args{"1001", "999"}, false, &corev1.SecurityContext{
-			RunAsUser:  pointInt64(1001),
-			RunAsGroup: pointInt64(999),
+		{"valid security context", args{"1001", "999", "true"}, false, &corev1.SecurityContext{
+			RunAsUser:                pointInt64(1001),
+			RunAsGroup:               pointInt64(999),
+			AllowPrivilegeEscalation: pointBool(true),
```

Comment:
Can you please switch this to use https://github.com/vmware-tanzu/velero/blob/main/pkg/util/boolptr/boolptr.go and remove the function you added on line 81?

Reply:
Fixed in 2342be1 and d29aa65.